src/types/Filter.ts

```diff
@@ -0,0 +1,10 @@
+export enum SortField {
+  All = 'all',
+  Active = 'active',
+  Completed = 'completed',
+}
+
+export interface IQuery {
+  status: string;
```

Comment:
why the `status` is string and not `SortField`?

Reply:
I set the initial values
useState<IQuery>({ status: 'all', query: '' });

Reply:
> useState({ status: 'all', query: '' });

`'all'` is one of  `SortField` enum values, so it makes sense to define `status` type as `SortField`.
Also, you should use enum for referring to value: `status: SortField.All` instead of `status: 'all'`.